app/assets/src/components/views/discovery/DiscoveryView.jsx

```diff
@@ -159,20 +175,30 @@ class DiscoveryView extends React.Component {
       "filters",
       "mapSidebarTab",
       "projectId",
+      "sampleActiveColumns",
       "search",
       "showFilters",
       "showStats",
     ];
     const stateFields = concat(urlFields, ["project"]);
+    const localFields = [
+      "currentTab",
+      "sampleActiveColumns",
+      "showFilters",
+      "showStats",
+    ];
 
     const historyState = pick(stateFields, this.state);
     const urlState = pick(urlFields, this.state);
+    const localState = pick(localFields, this.state);
 
+    // Saving on URL enables sharing current view with other users
     let urlQuery = this.urlParser.stringify(urlState);
     if (urlQuery) {
       urlQuery = `?${urlQuery}`;
     }
 
+    // History state may include some fields that enable fast loading of previous pages (e.g. project)
```

Comment:
this comment is a little confusing to me. "fast loading" as in direct loading of the previous pages?

Reply:
Loading when clicking back or the history dropdown on the top-left corner of the browser.

Reply:
Any suggestions for better wording?

Reply:
resolved